knowledge_storm/rm.py

```diff
@@ -1,7 +1,7 @@
 import logging
 import os
 from typing import Callable, Union, List
-
+from duckduckgo_search import DDGS
```

Comment:
Could we move this to `DuckDuckGoSearchRM.__init__()`? In this way, people don't necessarily need to install this package and we don't need to change `requirements.txt`.  You can do something similar to [here](https://github.com/stanford-oval/storm/blob/53b7f1ece52c1315694bb16e689e69a654332768/knowledge_storm/lm.py#L247)